eggslist/users/api/views.py

```diff
@@ -270,3 +302,56 @@ class FavoriteUsersListAPIView(ListAPIView):
 
     def get_queryset(self):
         return User.objects.filter(followers__user=self.request.user)
+
+
+class ConnectStripeCreateAPIView(APIView):
+    """
+    Create Stripe connection with user account
+    """
+
+    permission_classes = (IsAuthenticated, IsVerifiedSeller)
+
+    def create_stripe_account(self, request):
+        account = stripe.Account.create(
+            type=settings.STRIPE_SELLERS_ACCOUNT_TYPE,
+            email=request.user.email,
+            metadata={"user.id": request.user.id},
+        )
+        try:
+            user_stripe_connection_model = models.UserStripeConnection.objects.create(
```

Comment:
Why is this variable called "user_stripe_connection_model"? Looks like it is not a model, but an instance of the model, so shouldn't this be "user_stripe_connection"?